Tasks/UseDotNetV2/usedotnet.ts

```diff
@@ -1,81 +1,88 @@
 "use strict";
 import * as path from 'path';
 
-import * as tl from 'azure-pipelines-task-lib/task';
-
+import * as taskLib from 'azure-pipelines-task-lib/task';
 import { DotNetCoreVersionFetcher } from "./versionfetcher";
+import { globalJsonFetcher } from "./globaljsonfetcher";
 import { VersionInstaller } from "./versioninstaller";
 import { Constants } from "./versionutilities";
 import { VersionInfo, VersionParts } from "./models"
 import { NuGetInstaller } from "./nugetinstaller";
 
 async function run() {
-    let packageType = tl.getInput('packageType') || "sdk";
-    let versionSpec = tl.getInput('version');
-    if (versionSpec) {
-        console.log(tl.loc("ToolToInstall", packageType, versionSpec));
-        let installationPath = tl.getInput('installationPath');
-        if (!installationPath) {
-            installationPath = path.join(tl.getVariable('Agent.ToolsDirectory'), "dotnet");
-        }
-        let includePreviewVersions: boolean = tl.getBoolInput('includePreviewVersions');
-
+    let packageType = taskLib.getInput('packageType') || "sdk";
+    let versionSpec = taskLib.getInput('version');
+    let installationPath = taskLib.getInput('installationPath');
+    if (!installationPath) {
+        installationPath = path.join(taskLib.getVariable('Agent.ToolsDirectory'), "dotnet");
+    }
+    let includePreviewVersions: boolean = taskLib.getBoolInput('includePreviewVersions');
```

Comment:
but it's simpler to understand what input's the task really use. I would recommend that all inputs are read as soon as it is possible. So the user also gets an error as fast as possible.

And the check of the value is in the `versionfetcher`. So that is okay I think.